benchmarks/web-serving/db_server/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM ubuntu:14.04
-MAINTAINER Nooshin Mirzadeh <nooshin.mirzadeh@epfl.ch>
+MAINTAINER Mark Sutherland <mark.sutherland@epfl.ch>
```

Comment:
The MAINTAINER directive has been deprecated. It has to be replaced by a LABLE like this:
```
LABEL maintainer="mark.sutherland@epfl.ch"
```

Reply:
👍